### ISSUE TITLE
Automatic document updates and bintray are skipped

### ISSUE THREAD

**Issue by namusyaka** [kubernetes/kompose#1354]
On Travis, seems that write access requires at least one approval. Also, bintray looks like not working properly.
```
4.66s$ ./script/sync-docs.sh
Agent pid 21313
Identity added: script/deploy_key (script/deploy_key)
Cloning into 'kompose'...
Warning: Permanently added the RSA host key for IP address '140.82.113.3' to the list of known hosts.
Switched to a new branch 'gh-pages'
Branch 'gh-pages' set up to track remote branch 'gh-pages' from 'origin'.
Adding Jekyll file format to index.md
Adding Jekyll file format to architecture.md
Adding Jekyll file format to conversion.md
Adding Jekyll file format to development.md
Adding Jekyll file format to getting-started.md
Adding Jekyll file format to installation.md
Adding Jekyll file format to integrations.md
Adding Jekyll file format to maven-example.md
---
setup.md already contains Jekyll format
Adding Jekyll file format to user-guide.md
[gh-pages 678fe553] Update docs
 5 files changed, 32 insertions(+), 222 deletions(-)
 mode change 100644 => 100755 docs/user-guide.md
remote: error: GH006: Protected branch update failed for refs/heads/gh-pages.        
remote: error: 2 of 2 required status checks are expected. At least 1 approving review is required by reviewers with write access.        
To github.com:kubernetes/kompose.git
 ! [remote rejected]   gh-pages -> gh-pages (protected branch hook declined)
error: failed to push some refs to 'git@github.com:kubernetes/kompose.git'
Skipping a deployment with the bintray provider because this is not on the required runtime
```

**Comment by hangyan** [comment]
fixed by #1355 ?

**Comment by namusyaka** [comment]
Nope, I just ignored bintray. The automatic document updates are still failing because at least one approval is required to merge the commit generated by CI.
